ProgressBar/.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'airbnb',
+    'plugin:prettier/recommended',
+    'prettier',
+  ],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 13,
+    sourceType: 'module',
+  },
+  plugins: ['react', 'prettier'],
+  rules: {
```

Comment:
```suggestion
   overrides: [
    {
      files: ['*.js', '*.jsx'],
      rules: {
        semi: ['error', 'always'],
        quotes: ['error', 'single'],
        'no-eval': ['error', { allowIndirect: true }],
        'react/jsx-uses-vars': 'error',
        'no-console': 'error',
      },
    },
  ],
```
Иначе линтинг не чекает jsx файлы. У меня локально сразу линтинг ругался на App.jsx файл, но не фиксил autofixable issues при запуске. + Прогоняй его перед тем как коммитить